expression/integration_test.go

```diff
@@ -4611,3 +4611,13 @@ func (s *testIntegrationSuite) TestDatetimeMicrosecond(c *C) {
 		testkit.Rows("2007-03-28 22:08:27.999998"))
 
 }
+
+func (s *testIntegrationSuite) TestNotExistFunc(c *C) {
+	tk := testkit.NewTestKit(c, s.store)
+	_, err := tk.Exec("SELECT xxx(1)")
+	c.Assert(err.Error(), Equals, "[expression:1305]FUNCTION xxx does not exist")
```

Comment:
What will MySQL return if database is empty?

Reply:
MySQL returns "ERROR 1046 (3D000): No database selected".  I will fix this in this PR.